pageserver/src/tenant/delete.rs

```diff
@@ -322,9 +322,15 @@ impl DeleteTenantFlow {
         // Though sounds scary, different mark name?
         // Detach currently uses remove_dir_all so in case of a crash we can end up in a weird state.
         if let Some(remote_storage) = &remote_storage {
-            create_remote_delete_mark(conf, remote_storage, &tenant.tenant_shard_id)
-                .await
-                .context("remote_mark")?
+            create_remote_delete_mark(
+                conf,
+                remote_storage,
+                &tenant.tenant_shard_id,
+                // Can't use tenant.cancel, it's already shut down.  TODO: wire in an appropriate token
```

Comment:
which token would be appropriate here? the connection level token? or the ps shutdown token?

Reply:
The process-wide one -- i have a followup here that adds a cancellation token on the top level of TenantManager, and does the refactoring required to move the tenant deletion flow into TenantManager to use it https://github.com/neondatabase/neon/pull/6105